crates/biome_cli/src/commands/check.rs

```diff
@@ -74,13 +77,35 @@ pub(crate) fn check(
         session.app.console,
         cli_options.verbose,
     )?;
+    let (editorconfig, editorconfig_diagnostics) = {
+        let search_path = loaded_configuration
+            .directory_path
```

Comment:
I wonder what will happen if `.editorconfig` is in a deeper path from where the `biome.json` is. Because here we start searching from the path where the `biome.json` is.

Reply:
Okay, I will marked this as resolved as it is not in the scope of this PR.